addons/project/wizard/project_delete_wizard.py

```diff
@@ -10,14 +10,18 @@ class ProjectDelete(models.TransientModel):
 
     project_ids = fields.Many2many('project.project', string='Projects')
     task_count = fields.Integer(compute='_compute_task_count')
+    projects_archived = fields.Boolean(compute='_compute_projects_archived')
+
+    def _compute_projects_archived(self):
```

Comment:
Even we are in a wizard, it's better to make this compute multi (like `_compute_task_count`)